core/tests/protractor_utils/CommunityDashboardTranslateTextTab.js

```diff
@@ -23,11 +23,15 @@ var CommunityDashboardTranslateTextTab = function() {
   var selectableLanguageElements = element(
     by.css('.protractor-test-language-selector'));
   var selectedLanguageElement = selectableLanguageElements.element(
-    by.css('option:checked'));
+    by.css('.oppia-translation-language-selector-inner-container'));
```

Comment:
Do not use `oppia-` classes here, create a separate `protractor-test-` prefix class.

Reply:
Done